docs/initiatives/consent_manager/api_reference/index.md

```diff
@@ -0,0 +1,446 @@
+---
+sidebar_position: 3
+sidebar_label: API Reference
+---
+# Consent Manager - API Reference
+This section highlights the key apis accessible along with their capabilites, request body and appropriate response.
+
+## Authentication
+All requests to the Consent Manager APIs must include an authentication token in the request headers. To authenticate, add an `Authorization` header with the value `Bearer YOUR_TOKEN` in your HTTP requests.
+
+## Endpoints
+
+### Register a new Consent Artifact
+This allows to create a new consent artifact and associate it with a user.
+
+**Endpoint**: `/auth/register`
+
+**HTTP Method**: POST
+
+**Request Headers**:
+- `Content-Type: application/json`
+- `Authorization: Bearer YOUR_TOKEN`
+
+**Request Body**:
+
+```json
+{
+    "consentArtifact": {
```

Comment:
Kindly add a table below to explain each of these elements or do it as we have done for the Beckn loan spec doc as a table with 2 columns. Ref: https://github.com/beckn/protocol-specifications/discussions/363#discussioncomment-6508627